app/javascript/components/miq-alert-set-form/miq-alert-set-form.schema.js

```diff
@@ -51,13 +51,16 @@ function createSchema(fieldss, recordId, emsId, mode, loadSchema, alertState, al
               name: 'alert_profile_alerts',
               key: `alerts-${emsId}`,
               label: __('Alert Selection'),
-              leftTitle: emsId ? changeTitle(emsId, mode) : 'Avaliable Alerts',
+              leftTitle: emsId ? changeTitle(emsId, mode) : __('Avaliable Alerts:'),
               rightTitle: __('Profile Alerts:'),
               allToRight: false,
               moveLeftTitle: __('Remove'),
               moveAllLeftTitle: __('Remove All'),
               moveRightTitle: __('Add'),
               moveAllRightTitle: __('Add All'),
+              noValueTitle: __('No option selected'),
```

Comment:
I looked high and low for the Filter Value parameter to translate it can i couldn't find it, the DDF documentation [here](https://data-driven-forms.org/provided-mappers/dual-list-select?mapper=carbon&schema=basic) implies that that should be `filterValueTitle` but i've tried that and it didn't update the placeholder value. I looked into the other parameters to see if i could figured out which pointed to the placeholder but none of them worked. This is an issue with all the other placeholders for the Dual List Select